tests/components/context_menu.test.ts

```diff
@@ -1,3 +1,9 @@
+jest.mock("../../src/registries/menus/menu_items_actions", () => {
+  return {
+    ...jest.requireActual("../../src/registries/menus/menu_items_actions"),
+    ...jest.requireActual("../__mocks__/clipboard_item"),
+  };
+});
```

Comment:
I don't think you need to mock the file like this since you don't mock any function of the file.

You can mock the clipboard like we mocked the charts (see `mockChart`). It's easier to read.